### DISCUSSION TITLE
How does ngrinder write files through Jython script? Is the file in the resource directory

### DISCUSSION THREAD

**Discussion by zhougy1989** [naver/ngrinder#868] [Q&A]
How does ngrinder write files through Jython script? Is the file in the resource directory
![image](https://user-images.githubusercontent.com/50607317/151271040-ef7d79a5-d387-4d86-8ce5-f46e86c2c4c6.png)
i use this method ,but it not work
@junoyoon

**Comment by donggyu04** [comment]
@zhougy1989 

Do you have any logs?
`NgrinderSecurityManager` may have blocked this operation.
